parameters.yaml

```diff
@@ -0,0 +1,32 @@
+# parameters.yaml
+
+# Parameters file format version 0.1
+# This is a draft YAML file for data in a single experiment run
+# Subject to changes as needed
+
+paths:
+    base: <To be set to the local computer's data directory>
+    protocol: HB_139_DAMGO/
+    unit: 825_12h24m37s/20200825_12h24m37s_units_ts.mat
```

Comment:
maybe "neuron_unit" instead?

Reply:
That is more clear to use neuron. In the field of electrophysiology, "unit" is often used as short hand for "putative neuron" because the signal we assume is a neuron, but ultimately, we don't always know for sure. "Unit" is less explicitly claiming to be a neuron. It is a "unit" of repeated activity in the time series that is under examination as a putative neuron.